google/cloud/bigtable/testing/inprocess_data_client.cc

```diff
@@ -69,6 +69,14 @@ InProcessDataClient::ReadRows(grpc::ClientContext* context,
   return Stub()->ReadRows(context, request);
 }
 
+std::unique_ptr<::grpc::ClientAsyncReaderInterface<btproto::ReadRowsResponse>>
+InProcessDataClient::AsyncReadRows(
+    ::grpc::ClientContext* context,
+    const ::google::bigtable::v2::ReadRowsRequest& request,
```

Comment:
I see a mix of west-const and east-const. West-const here, and east const in the signature 7 lines above. I tend to prefer west-const in general for reasons ;-), but I mostly prefer consistency. May want to change this to east-const to match the surrounding code more, or change other cases to west-const. I'm not really placing a vote here, just an observation.